### ISSUE TITLE
Unnecessary `./custom` directory

### ISSUE THREAD

**Issue by ddejong-therp** [sunflowerit/waftlib#43]
Hey,

I generally try not to comment on naming conventions too much, but at the moment the waft build uses the `./custom/src` directory for both the source repositories, and the addons.yaml & repos.yaml files. However, the `custom` directory contains nothing other than the `src` directory, so it seems unnecessary to have this directory structure.

Why not place everything in the `./custom/...` or `./src/...` directory? Or otherwise, maybe it also makes sense to put the repos.yaml and addons.yaml in custom, and have all the repo directories be in `./custom/src`. But personally I prefer a single directory, because it is a directory we often find ourselves navigating too.

**Comment by thomaspaulb** [comment]
I agree, and actually the new version of Waft by Hussam changes the whole directory structure to something more simple. The custom directory is a leftover from Doodba, where it made sense for Docker-technical reasons.

Therp - Open Source ERP solutions built on Odoo
